session_rpcserver.go

```diff
@@ -190,7 +190,7 @@ func (s *sessionRpcServer) AddSession(ctx context.Context,
 	req *litrpc.AddSessionRequest) (*litrpc.AddSessionResponse, error) {
 
 	expiry := time.Unix(int64(req.ExpiryTimestampSeconds), 0)
```

Comment:
I think this one should also be:
 
```suggestion
	expiry := time.Unix(int64(req.ExpiryTimestampSeconds), 0).UTC()
```

Or we need to update protos to declare that the timestamp should explicitly be sent in through the request in UTC?